web/src/app/modules/views/main/editors/modules/contents-container/services/content-container.service.ts

```diff
@@ -0,0 +1,18 @@
+import { EventEmitter, Injectable } from '@angular/core';
+import { IContainer } from '../../../../../../../model/IContainer';
+import { Subject } from 'rxjs';
+
+ @Injectable()
+export class ContentsContainerService {
+
+   private onModelDeleted = new Subject<string>();
+
+   deleted$ = this.onModelDeleted.asObservable();

```

Comment:
There is no visibility annotated to deleted. Additionally, we do not use the $-postfix notation (since we actually use EventEmitters for this purpose).